src/pages/Login/Login.jsx

```diff
@@ -61,14 +85,39 @@ export const Login = () => {
 					}
 				</div>
 
+				<button
+					onClick={() =>
+						setCredentials({
+							email: "adarshbalika@gmail.com",
+							password: "adarshbalika",
+						})
+					}
+					className="p-1 w-100 font-semibold btn btn-outlined transition-2 mr-1 mb-2 rounded-sm"
+				>
+					Login using dummy credentials
+				</button>
+
 				<button className="p-1 w-100 font-semibold btn btn-solid transition-2 mr-1 mb-2 rounded-sm">
 					Login
 				</button>

```

Comment:
what does this button do?

Reply:
For the login form, there are two buttons, if the user clicks "Login with dummy credentials" then it will use that credentials, the other login button is for normal login, any one of them is clicked then it will trigger form submit because the form has a behavior where if any button clicked inside a form will trigger the submit event.